jobbergate-api/pyproject.toml

```diff
@@ -106,6 +107,7 @@ module = [
     "uvicorn",
     "asyncpg.*",
     "yaml",
+    "toml",
```

Comment:
is toml is already a dependency then it doesn't need to be a dev depency

Reply:
The line you commented on is not for dev dependencies, but for `tool.mypy.overrides`, so I think we are good.